core/mbox.c

```diff
@@ -125,3 +125,30 @@ int _mbox_get(mbox_t *mbox, msg_t *msg, int blocking)
         return 0;
     }
 }
+
+int mbox_try_get_with_type(mbox_t *mbox, msg_t *msg, uint16_t type)
+{
+    unsigned irqstate = irq_disable();
+    unsigned end_pos = mbox->cib.read_count + cib_avail(&mbox->cib);
+    const unsigned mask = mbox->cib.mask;
+    for (unsigned pos = mbox->cib.read_count; pos != end_pos; pos++) {
```

Comment:
```suggestion

    for (unsigned pos = mbox->cib.read_count; pos != end_pos; pos++) {
```

I agree with uncrustify